src/main/scala/tech/cryptonomic/conseil/tezos/NetworkConfigOperations.scala

```diff
@@ -0,0 +1,20 @@
+package tech.cryptonomic.conseil.tezos
```

Comment:
This functionality is generic, i.e. applies for Tezos as well as any other chains we support, so it should not be in the `tezos` package.

Reply:
ok, moved